src/components/Button/__snapshots__/Button.test.tsx.snap

```diff
@@ -85,7 +85,7 @@ exports[`Button Component renders button with icon right correctly 1`] = `
 exports[`Button Component renders circle button correctly 1`] = `
 <DocumentFragment>
   <button
-    class="ant-btn css-dev-only-do-not-override-1vr7spz ant-btn-circle ant-btn-primary button"
+    class="ant-btn css-dev-only-do-not-override-1vr7spz ant-btn-circle ant-btn-primary button buttonMdIconOnly"
```

Comment:
Why has this class appeared? Also why in camelCase instead of the typical kebab-case?

Reply:
It is a remnant of the PR on Button, where I had forgotten to update the snapshots (we should configure the pipeline so that it blocks the merge operation if the pipeline fails). The use of camelCase is due to the fact that it is a class set by us via CSS, and with the use of CSS modules it is more convenient (there have already been discussions about it in the PR on Button)